crates/dns-vcr/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "dns-vcr"
```

Comment:
Not a big fan of the name? Would suggest something more direct like `dns-recorder` or `dns2json` which doesn't rely on a decade-old device metaphor acronym.